docs/reference/remap.cue

```diff
@@ -52,17 +51,589 @@ remap: {
 			return: [#RemapReturnTypes, ...#RemapReturnTypes]
 			category:    "coerce" | "numeric" | "object" | "parse" | "text" | "hash" | "event" | "networking"
 			description: string
-			examples: [#RemapExample, ...#RemapExample]
+			examples?: [#RemapExample, ...#RemapExample]
 			name: Name
 		}
 	}
 
 	errors: {
 		ArgumentError: {
-			description: "Raised when the provided input is not a supported type."
+			description: "Raised when the provided input is not a supported type for that function."
 		}
 		ParseError: {
 			description: "Raised when the provided input cannot be parsed."
 		}
 	}
```

Comment:
This is left over from previous work. I just updated the wording. This will almost certainly change in the next few weeks so I'll leave it as-is for now.